docs/cli.rst

```diff
@@ -48,6 +51,12 @@ Or with a filename::
     export FLASK_APP=/path/to/hello.py
     flask run
 
+If you use Flask 0.13 or later you can also set the environment variables
+in a ``.env`` file.  For instance like this::
+
+    FLASK_APP=/path/to/hello.py
+    FLASK_DEBUG=1
+
```

Comment:
Imagine two flask apps using this that are installed in the same environment - how would you know which one to use? Also, I think for bigger applications it's nice to have a custom command named like the application without losing any of the `flask` command's benefits.